client/my-sites/plans-features-main/index.tsx

```diff
@@ -679,6 +679,18 @@ const PlansFeaturesMain = ( {
 		? getWooExpressFeaturesGroupedForFeaturesGrid()
 		: getPlanFeaturesGroupedForFeaturesGrid();
 
+	const getComparisonGridToggleLabel = () => {
+		if ( showPlansComparisonGrid ) {
+			return translate( 'Hide comparison' );
+		}
+		switch ( flowName ) {
+			case ONBOARDING_GUIDED_FLOW:
```

Comment:
For a more general solution, can we use `if ( gridPlansForFeaturesGrid.length < gridPlansForComparisonGrid.length ) ` instead?

Reply:
@alshakero - Can you elaborate a bit on this? Whats the purpose of the `gridPlansForFeaturesGrid.length < gridPlansForComparisonGrid.length` comparison?

Reply:
The button needs to say "Show all plans" when the PlansForFeaturesGrid has fewer plans than the PlansForComparisonGrid. As in, the features grid is showing **_some_** plans, do you want **_all_** plans? 

This translates to `gridPlansForFeaturesGrid.length < gridPlansForComparisonGrid.length`